lbrynet/dht/contact.py

```diff
@@ -110,10 +113,13 @@ def __ne__(self, other):
         else:
             return True
 
+    def __hash__(self):
```

Comment:
why this?

Reply:
also, why sometimes the id and sometimes a sum made from the address? doesn't seem like a good key if it's being used in a dictionary.